R/packageSetup.R

```diff
@@ -0,0 +1,85 @@
+# Create readme, update readme on new project status
+
+
+#' generates a ReadMe.md file in the project's home directory. Uses readme_template.md.
+#' Asks for user's input
+#' @export
+addReadMe <- function(){
+
+  readline(prompt="You're going to be asked a few questions about the study.
```

Comment:
think there are too many prompts here. It seems to be cumbersome to the user and what if they do not have the input on hand at that point in time.

I would like it to use dots for parameter inputs

addReadMe(...)